tool/travis_flutter_script.sh

```diff
@@ -48,6 +48,9 @@ do
     echo "== Testing '${PROJECT_NAME}' on Flutter's ${FLUTTER_VERSION} channel =="
     pushd "${PROJECT_NAME}"
 
+    # Grab packages.
+    "${LOCAL_SDK_PATH}/bin/flutter" pub get
```

Comment:
Weird that `flutter analyze` runs `pub get` on a fresh clone for me. 

It's probably better not to rely on this behavior anyway, since it's not clear what criteria it's using.

Reply:
Weird. Now that I've double-checked it, I'm getting the same result as you. My local copy must have somehow gotten into a weird state, which I now can't reproduce.

> It's probably better not to rely on this behavior anyway, since it's not clear what criteria it's using.

Agreed.